crates/control/src/behavior/fall_safely.rs

```diff
@@ -1,8 +1,10 @@
 use types::{FallState, MotionCommand, WorldState};
 
-pub fn execute(world_state: &WorldState) -> Option<MotionCommand> {
-    match world_state.robot.fall_state {
-        FallState::Falling { direction } => Some(MotionCommand::FallProtection { direction }),
+pub fn execute(world_state: &WorldState, has_ground_contact: &bool) -> Option<MotionCommand> {
```

Comment:
Why is this a reference to a boolean instead of just a boolean?

Reply:
Done